src/brushes/pencil_brush.class.js

```diff
@@ -272,8 +255,8 @@
       if (this.decimate) {
         this._points = this.decimatePoints(this._points, this.decimate);
       }
-      var pathData = this.convertPointsToSVGPath(this._points).join('');
-      if (pathData === 'M 0 0 Q 0 0 0 0 L 0 0') {
+      var pathData = this.convertPointsToSVGPath(this._points);
+      if (this._isEmptySVGPath(pathData)) {
```

Comment:
Maybe there's a better way to determine if the path is empty.
What about checking points?

Reply:
Probably there is, but for now we don't care. we know this is a bit bad because we need to join it all of it to check if is empty. If is a 20 meter long path with 10.000 points is time wasted. But for now we don't care. for this PR is fine as it is.